### DISCUSSION TITLE
Not running between batches

### DISCUSSION THREAD

**Discussion by siggipop** [Innofactor/XrmToolBox.BulkDataUpdater#181] [General]
I have experienced with the latest version that between batch updates, the application stops updating i.e. doesn't run next batch updates automatically. It's like it thinks it's done with all work. 

I update with batch size off 200, and wait 5 seconds between calls, and update 2 fields to NULL

**Comment by rappen** [comment]
_Sorry late response, I must have missed the notification..._

Do you know if this started to appear in latest version (1.2022.11.2)?
Still the problem?
Always?

**Comment by siggipop** [comment]
No - I wasn't aware if this appeared in a specific version (There can be some while between me using the functionality and therefore i cannot be sure when/which version introduced the change) 

But! This happened som time ago, and later I recalled the issue was gone, and i couldn't reproduce it any longer!
I regret that I (even if I had intended to ) forgot to update this thread with my findings. 

I'm mostly convinced now, the issue was locally on my machine and at the specific moment in time. So maybe a cach/runtime issue!?

**Reply by rappen** [reply]
Thanks - let me know if it appears again.
